### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by BartDelrue** [sachinchoolur/lightgallery.js#79]
There are some accessibility issues I'd like to tackle to make lightgallery WCAG 2.0 compliant.
- Previous, next, close and all module 'buttons' are not buttons. Keyboard users cannot use them.
- The tabfocus does not loop over the visible elements, but continues looping over all focusable elements on the -now invisible- page.

I've had a look at the code and I could create a tab trap to take care of the focusable elements.
Unfortunately all module 'buttons' are added to the DOM async each time the gallery is opened
This means you have to collect all focusable elements each keypress event, just to make sure all buttons have been added. I'd rather just do it once during initialization and store them.
Would it be an option to add a callback to all module init functions?

**Comment by niemenmaa** [comment]
Any updates on this? WCAG 2.0 compliance would be so nice for this awesome library!

**Comment by darylknight** [comment]
+1. I might have to strip this out as it's currently not usable with a keyboard at all once you open an image
